CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+## 1.8.0 / 2017-10-06
+
+* [FEATURE] Ability to act as a remote read server for other Prometheus servers.
+* [FEATURE] K8s SD: Support discovery of ingresses.
+* [FEATURE] Consul SD: Support for node metadata.
+* [FEATURE] Openstack SD: Support discovery of hypervisors.
+* [FEATURE] Expose current Prometheus config via `/status/config`.
+* [FEATURE] Allow to collapse jobs on `/targets` page.
+* [FEATURE] Add `/-/healthy` and `/-/ready` endpoints.
+* [FEATURE] Add color scheme support to console templates.
+* [CHANGE] Rule links link to the _Console_ tab rather than the _Graph_ tab to
+  not trigger expensive range queries by default.
+* [CHANGE] Changed help link to `https://prometheus.io/docs`.
+* [ENHANCEMENT] Remote storage connections use HTTP keep-alive.
+* [ENHANCEMENT] Improved logging about remote storage.
+* [ENHANCEMENT] Failure to send NaN values to remote storage logged on debug
+  level only.
+* [ENHANCEMENT] Improved URL validation, including not panic'ing if invalid
+  URLs are provided on the command line.
```

Comment:
That was actually the case before already. We actually relaxed the validation a bit now, as we'll now allow underscores in domain names. The intention of the change was to remove the govalidator dependency.

Reply:
There were certain URLs that caused a panic in 1.7 (starting with a quote).

"Improved validation" does include more relaxed validation. I didn't want to provide all the details (what exactly got relaxed and what is validated more strictly now) as that's not the point of the changelog.

Reply:
After personal communication, I understood that the panic was _introduced_ post-1.7 and then also fixed, but that doesn't have to be reflected in the changelog, obviously.